templates/docs/examples/patterns/notifications/borderless.html

```diff
@@ -24,7 +24,7 @@
       <td colspan="4">
         <div class="p-notification--caution is-borderless u-no-margin--bottom">
```

Comment:
@lyubomir-popov I know it's not your change, but I noticed this one relies on `u-no-margin--bottom`.

Do we want it to have bottom margin, or remove it completely from this variant.
Or, if we need utility in the example, we need to add it to standalone stylesheet.

Reply:
we do need it. please add it to the standalone filesif that's ok

Reply:
Ok, will do when I remove the other variants.